vanilla_first_setup/utils/processor.py

```diff
@@ -135,19 +127,7 @@ def hide_first_setup(user: str=None):
         if user is None:
             user = os.environ.get("USER")
 
-        desktop_file = "/home/%s/.local/share/applications/org.vanillaos.FirstSetup.desktop" % user
         autostart_file = "/home/%s/.config/autostart/org.vanillaos.FirstSetup.desktop" % user
```

Comment:
You moved the autostart file to `/etc`, so this needs to be changed as well

Reply:
Based on the code, this autostart file is the file created when first-setup is installed, which is in `~/.config/autostart/org.vanillaos.FirstSetup.desktop`

Reply:
You're right, my bad